AWS/CDK/lib/lambda-stack.js

```diff
@@ -0,0 +1,197 @@
+const cdk = require('@aws-cdk/core');
+const s3 = require('@aws-cdk/aws-s3');
+const dynamodb = require('@aws-cdk/aws-dynamodb');
+const apigw = require('@aws-cdk/aws-apigateway');
+const iam = require('@aws-cdk/aws-iam');
+const lambda = require('@aws-cdk/aws-lambda');
+const { RestApi, MethodLoggingLevel } = require('@aws-cdk/aws-apigateway');
+const { PolicyStatement } = require('@aws-cdk/aws-iam');
+
+class LambdaApiStack extends cdk.Stack {
+  /**
+   * Constructor
+   * @param {cdk.Construct} scope
+   * @param {string} id
+   * @param {cdk.StackProps=} props
+   */
+  constructor(scope, id, props) {
+    super(scope, id, props);
+
+    this.createDynamoTables();
+    this.createLambdaExecutionRole();
+    this.createLambdaFunctions();
+    this.createApi();
+  }
+
+  dynamoTableArns() {
+    return [
+      this.tblDevices.tableArn,
+      this.tblInfractions.tableArn
+    ];
+  }
+
+  createDynamoTables() {
+    this.tblDevices = new dynamodb.Table(this, 'DynamoDBDevices', {
+      partitionKey: {
+        name: 'id',
+        type: dynamodb.AttributeType.STRING
+      },
+      readCapacity: 5,
+      writeCapacity: 5,
+      removalPolicy: cdk.RemovalPolicy.DESTROY
+    });
+
+    this.tblInfractions = new dynamodb.Table(this, 'DynamoDBInfractions', {
+      partitionKey: {
+        name: 'id',
+        type: dynamodb.AttributeType.STRING
+      },
+      readCapacity: 5,
+      writeCapacity: 5,
```

Comment:
5?

Reply:
So if you use provisioned IOPS you have to set this.  5 is a reasonable # of IOPS that will cause your dynamodb to throttle if traffic becomes too heavy.  For real use cases you might use request billing.